### ISSUE TITLE
Locale Issue

### ISSUE THREAD

**Issue by recom273** [mikebrady/shairport-sync#790]
Not really an issue, as the install seemed to work out, but I got a repeated perl warning today while installing.

[pi@shairport:~ $ git clone https://github.com/mikebrady/shairport-sync.git
Cloning into 'shairport-sync'...
remote: Enumerating objects: 131, done.
remote: Counting objects: 100% (131/131), done.
remote: Compressing objects: 100% (94/94), done.
remote: Total 11883 (delta 69), reused 80 (delta 37), pack-reused 11752
Receiving objects: 100% (11883/11883), 5.72 MiB | 1.54 MiB/s, done.
Resolving deltas: 100% (8034/8034), done.
pi@shairport:~ $ cd shairport-sync
pi@shairport:~/shairport-sync $ autoreconf -i -f
perl: warning: Setting locale failed.
perl: warning: Please check that your locale settings:
	LANGUAGE = (unset),
	LC_ALL = (unset),
	LC_CTYPE = "UTF-8",
	LANG = "en_GB.UTF-8"
    are supported and installed on your system.
perl: warning: Falling back to a fallback locale ("en_GB.UTF-8").
perl: warning: Setting locale failed.
perl: warning: Please check that your locale settings:
	LANGUAGE = (unset),
	LC_ALL = (unset),
	LC_CTYPE = "UTF-8",
	LANG = "en_GB.UTF-8"
    are supported and installed on your system.
perl: warning: Falling back to a fallback locale ("en_GB.UTF-8").
perl: warning: Setting locale failed.

It continued over several sections, I left the locale settings as unchanged when loading Raspian.

**Comment by mikebrady** [comment]
Thanks for the post. I’ve not seen this error, but then I always set the locale when I install a fresh Raspbian. I use the raspi-configura tool.

Is your Raspbian reasonably up-to-date?

**Comment by recom273** [comment]
Yes, new version of raspbian November 2018

I just ran the setup wizard, on first instal I changed the setup to my time zone, before sudo update and upgrade, and I got the error. 

I thought I would see if I could get a clean shairport install so, so after reinstalling raspian and the initial boot up, I didn’t change the locale settings which were as shown. 

I hadn’t experienced this before, this was my 2nd setup, the first time, it was effortless. I run the instructions from pimylifeup.com

It doesn’t effect the function, I can see the endpoint in iTunes, Although I have yet run it, but I anticipate no problems.

Thanks anyway

**Comment by mikebrady** [comment]
Well, Shairport Sync doesn't use `perl` or locale information when it's running, so I guess that it should be okay. 

I have just run though the installation of Raspbian Lite via the command line, using `raspi-config` and have set the time zone and a different locale. It asked for a default locale, and I set that to my new locale. At that point, before rebooting, `perl` reported problems:

```
perl: warning: Setting locale failed.
perl: warning: Please check that your locale settings:
	LANGUAGE = (unset),
	LC_ALL = (unset),
	LANG = "en_GB.UTF-8"
    are supported and installed on your system.
perl: warning: Falling back to the standard locale ("C").
```
When I rebooted, everything was fine. Does this seem like your experience?
